client/dist/index.html

```diff
@@ -12,7 +12,7 @@
       <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
       <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
     </head>
-    <body>
+    <body style="background-color: #f7f7f7;">
```

Comment:
Instead of having inline style on this html file, perhaps we should move the existing `#app` background-color over to `body` in `styles.css`?

https://github.com/Audacious-Alder/FEC/blob/main/client/dist/style.css#L3

Reply:
fixed in most recent commit

Reply:
Thanks! Would you mind removing the background color from `index.html` (line 15) and `style.css` (line 3) since those should be redundant now?